.travis.yml

```diff
@@ -1,14 +1,10 @@
-sudo: required
-dist: xenial
+os: linux
+dist: focal
 services:
   - docker
-language: scala
-scala:
-  - 2.12.9
+language: minimal
```

Comment:
might be easier to discuss these in a tech talk... 🙂

Reply:
TL;wasn't at the discussion:

This PR switches to minimal CI images and then downloading small known quantity packages consistently across ALL CI system, versus requesting various CI images of "something scala" that were bundled with PostgreSQL and MySQL and all kinds of other stuff that needed to be turned off.